src/Claimer.sol

```diff
@@ -1,70 +1,102 @@
 // SPDX-License-Identifier: UNLICENSED
 pragma solidity ^0.8.17;
 
+// import "forge-std/console2.sol";
+
 import { SD59x18 } from "prb-math/SD59x18.sol";
 import { UD2x18 } from "prb-math/UD2x18.sol";
+import { UD60x18 } from "prb-math/UD60x18.sol";
 import { PrizePool } from "v5-prize-pool/PrizePool.sol";
+import { Multicall } from "openzeppelin/utils/Multicall.sol";
 
 import { LinearVRGDALib } from "./lib/LinearVRGDALib.sol";
 import { IVault } from "./interfaces/IVault.sol";
 
-contract Claimer {
+struct Claim {
+    IVault vault;
+    address winner;
+    uint8 tier;
+}
+
+contract Claimer is Multicall {
+
+    error DrawInvalid();
 
     PrizePool public immutable prizePool;
+    UD2x18 public immutable maxFeePortionOfPrize;
```

Comment:
This value should be updatable no?
What if we want to reduce the fee portion for all claims?

Reply:
I don't think the fees would ever reach the max portion; it's really just a back-stop.

We want to avoid having any kind of admins / owners!